### PR TITLE
Fix the issue in TelephonyCollector for dual sim phones iOS15 and less

### PR DESCRIPTION
**Pull request by vahancouver**
# JIRA Ticket

[SDKS-2776](https://bugster.forgerock.org/jira/browse/SDKS-2776)

# Description

Fix the issue in TelephonyCollector for dual sim phones iOS15 and less

### CODE REVIEW

File: FRAuth/FRAuth/Device/TelephonyCollector.swift [hunk 1]
```diff
@@ -28,35 +29,53 @@ public class TelephonyCollector: DeviceCollector {
...
+        var carrier: CarrierInfo?
         
-        if #available(iOS 12.0, *) {
-            if let providers = networkInfo.serviceSubscriberCellularProviders, providers.keys.count > 0 {
-                for (_, thisCarrier) in providers{
-                    carrier = thisCarrier
-                    break
-                }
+        
+        if let providers = networkInfo.serviceSubscriberCellularProviders, providers.keys.count > 0 {
```
Comment (jeyanthanperiyasamy) [inline FRAuth/FRAuth/Device/TelephonyCollector.swift:35]: i think we can simplify this to one line may be ?

 var carriers = providers.map { (carrierName: $0.value.carrierName , isoCountryCode: $0.value.isoCountryCode ) }

**Review by jeyanthanperiyasamy — COMMENTED** [review]
overall looks good, just a minor comment

**Review by jeyanthanperiyasamy — APPROVED** [review]
we need to have a ticket to either stop supporting this in future versions to avoid inconsistency between ios versions

**Review by spetrov — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**